website/src/components/HomepageFeatures/index.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 Datafuse Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import Link from "@docusaurus/Link";
+
+const FeatureList = [
+  {
+    title: 'Access data freely',
+    Svg: require('@site/static/img/undraw_the_world_is_mine.svg').default,
```

Comment:
This SVG appears to be depicting a pregnant woman, which seems inappropriate. Therefore, I would prefer to replace it.

Reply:
Updated.